ocs_ci/ocs/resources/pod.py

```diff
@@ -298,6 +298,8 @@ def run_io(
         fio_filename=None,
         bs="4K",
         end_fsync=0,
```

Comment:
If we are not using direct IO and we are not using fsync, we can't be sure data has landed fully. I suggest using direct IO.